docs/move/overview-v2.md

```diff
@@ -0,0 +1,35 @@
+---
+id: overview-v2
```

Comment:
What will this be named when we finally go live? Is it going to stay `overview-v2`?

Reply:
The other one is called `overview`? Will that be named something else when we go live?

Reply:
For example:

This one will `overview-v2` => `overview`

and

the other one will go from `overview` => `introduction`

?

Reply:
We can change it to anything you want. This is on the list of things to go over after code freeze (this Friday) and before launch. Do you want me to make the changes you suggested in this thread?

Reply:
Yes. Please update as Joel mentions.